addons/Media_Recorder/src_webpack/MediaRecorder.jsm

```diff
@@ -768,19 +758,33 @@ export class MediaRecorder {
         }
     }
 
+    _executeNotification(notifyInput) {
+        try {
+            if (mLibroChromium != undefined) {
```

Comment:
Jeżeli będziesz jeszcze w tym ticketcie coś robił, to prosiłbym tutaj o komentarz skąd tą zmienną wziąłeś, ale dla samego komentarza nie ma co rozgrzebywać ticketu.